schedule/lvm_thin_provisioning.yml

```diff
@@ -0,0 +1,77 @@
+name:           lvm_thin_provisioning
+description:    >
+    Complete OS deployment with unencrypted LVM drive management.
+    Test creates 2 LVM and BIOS boot partitions.
+    Thin pool and thin lv resides on the second LVM partition,
+    where /home (XFS) is being mounted. Maintainer: mloviska, poo#39023
+conditions:
```

Comment:
what about to call it `branching`?

Reply:
Hmm, to be honest I guess both are not descriptive enough... I was thinking of `conditional_modules`, but that is maybe too long. WDYT?

Reply:
In my initial proposal https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6329/commits/83d3935535d9c96b04ea5c04b7269ac17f7f2cb6 I stated that the format is inspired by https://docs.gitlab.com/ee/ci/yaml/ where we would simply have in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6329/commits/83d3935535d9c96b04ea5c04b7269ac17f7f2cb6#diff-a21202331ee1025b4c7c79e798ded385R220 either "only" or "except"

Reply:
yeah, for which you had to introduce eval of perl code, because it doesn't allow us to express any of the things we have.

Current approach could be extended to include even complex conditions
```yaml
- "AND"
    - "var1" : "value1",
    -- "OR":
          "var2" : "value2"
          "var3" : "value3"
```
But I would say that's overkill which will bring us back to what we have, but in yaml.

Reply:
Nah, it was only about the naming, not the logic.